### ISSUE TITLE
"Couldn't get your external IP" Error

### ISSUE THREAD

**Issue by engineer22** [aaronjwood/PortAuthority#142]
It's been a week that I noticed the app can't get the WAN address. I also checked with other devices I have it installed and I get the same (tried on different networks too).

The connection seems to be successful, there is no log that I can't connect, so it must be app/server which connects to get the IP.

**Comment by aaronjwood** [comment]
Yeah, that's to be expected right now. The infra where I run this service is currently down. I should have it up next week sometime. Sorry about that

**Comment by engineer22** [comment]
Thanks for letting me know @aaronjwood, at least we now know why.
